helm/cmsweb/crabcache/values.yaml

```diff
@@ -106,25 +113,7 @@ serviceMon:
   protocol: TCP
   name: crabcache-mon
  
-command:
-- /bin/bash
-- /opt/setup-certs-and-run/setup-certs-and-run.sh
```

Comment:
@arooshap 

This command (` - /opt/setup-certs-and-run/setup-certs-and-run.sh`) should not be deleted. This is in main container.

Reply:
It wasnt deleted, it was shifted upwards.
![image](https://user-images.githubusercontent.com/77356768/130585774-08460aa8-3e9a-411b-97ef-38941391af49.png)